bindings/java/src/org/sleuthkit/datamodel/HostManager.java

```diff
@@ -285,22 +290,57 @@ public Host createHost(String name) throws TskCoreException  {
 			throw new TskCoreException(String.format("Error getting hosts"), ex);
 		}
 	}
-	
+
+	/**
+	 * Get all hosts with a particular status.
+	 *
+	 * @param status The status to filter on (if null, this will search for
```

Comment:
The status will never be null in the case database. Passing null for host status should cause an IllegalArgumentException.